### ISSUE TITLE
CONSIDER log output when platforms are installed

### ISSUE THREAD

**Issue by egnor** [robgonnella/ardi#63]
(This is EXTREMELY minor!)

There's log output when libraries are installed, but not platforms. So if you do an `ardi install`, you might get this:

```
$ ~/go/bin/ardi install
ardi    Installed library: Adafruit CAP1188 Library 1.0.2 
```

In reality it also installed the `arduino:avr` platform, which is why the command took as long as it did. This is mainly good for clarity, especially about running time.

**Comment by robgonnella** [comment]
Easy peasy!

**Comment by robgonnella** [comment]
Updated in new beta version, you can try it out via `GO111MODULE=on go get github.com/robgonnella/ardi/v2@v2.3.0-beta.1`

**Comment by robgonnella** [comment]
Closing: Addressed in v2.3.0
